### DISCUSSION TITLE
Combine selectors with AND

### DISCUSSION THREAD

**Discussion by MihaMarkic** [AvaloniaUI/Avalonia#8682] [Q&A] [answered]
AFAIK selectors allow only OR condition, but sometimes AND could be useful.
Imagine that a control has classes _pressed_ and _pointerover_, i.e. Button.
One might want to define different setters based on all four combinations of given classes, not just one or the another.
Or is there an existing way of handling such scenarios?

**Comment by Sorien** [comment]
not sure if it's implemented but it should work the same as in css just use `.` https://css-tricks.com/multiple-class-id-selectors/

**Comment by grokys** [comment, marked as answer]
I might be missing something, but a selector for button with both pressed and pointerover is simply:

```
Button:pressed:pointerover
```

**Reply by MihaMarkic** [reply]
Looks like what I'm after.
How does one write pressed when not pointerover?
a) something like Button:pressed:not(:pointerover) or 
b) simply Button:pressed and use correct order within styles I assume?

**Reply by maxkatz6** [reply]
It will work in either way

**Comment by MihaMarkic** [comment]
Thanks guys!
